react-form-controls/justin-app/src/RegistrationFormControlled.js

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+
+export default function RegistrationFormControlled() {
+  const [userName, setUserName] = useState(' ');
```

Comment:
Use an empty string for these instead of a space. With a space, the user will have to delete the space and start typing. Also, with a space, the "placeholder" text won't be displayed.